src/lib/import-export/import/importers/importer.ts

```diff
@@ -121,6 +121,7 @@ abstract class BaseBackupImporter extends BaseImporter {
 			await this.moveExistingDatabaseToTrash( dbPath );
 			await this.createEmptyDatabase( dbPath );
 			await this.importDatabase( rootPath, siteId, this.backup.sqlFiles );
+			await this.importWpConfig( rootPath );
```

Comment:
Let's create the wp-config file before `importDatabase` to fix https://github.com/Automattic/studio/pull/437#issuecomment-2286057665

```suggestion
			await this.importWpConfig( rootPath );
			await this.importDatabase( rootPath, siteId, this.backup.sqlFiles );
```